pkg/util/spanlogger/spanlogger.go

```diff
@@ -61,16 +56,5 @@ func FromContext(ctx context.Context) *SpanLogger {
 // within the context. If the context doesn't have a logger, the fallback
 // logger is used.
 func FromContextWithFallback(ctx context.Context, fallback log.Logger) *SpanLogger {
-	logger, ok := ctx.Value(loggerCtxKey).(log.Logger)
```

Comment:
Removed because `FromContext` checks the `ctx` for a logger to use and extracts the existing span (or uses a no-op). I'd appreciate another set of eyes here though to confirm my understanding is correct.